docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -206,7 +201,7 @@ If you need to use an integration that you have previously revoked, log in again
 
 ## Why does Codacy ask for permission to create SSH keys?
 
-Codacy needs to create an SSH key in your account to clone repositories and any associated git submodules.
+When you add a repository to Codacy, Codacy adds a new SSH key to your repository using an integration with your Git provider. Codacy uses this SSH key every time it needs to clone that repository.
 
 **Codacy only adds read-only SSH keys to be able to clone repositories** and won't have access to any of your existing SSH keys. You have full control over which organizations and repositories Codacy is authorized to access, and you can also [revoke the keys created by Codacy at any time](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-ssh-keys). Codacy doesn't change the contents or member privileges of any repository you authorize it to analyze.
```

Comment:
The current link points to a page related to account keys. Maybe we should link to this one instead?
[https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-deploy-keys](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-deploy-keys)

Also, we are pointing to GitHub documentation. What about Bitbucket and GitLab?